python/lsst/ts/idl/enums/Scheduler.py

```diff
@@ -28,3 +31,30 @@ class SalIndex(enum.IntEnum):
 
     MAIN_TEL = 1
     AUX_TEL = 2
+
+
+class DetailedState(enum.IntEnum):
+    """Detailed state enumeration for the Scheduler.
+
+    This enumeration is added here temporarily to support publishing this
```

Comment:
I think this comment should be removed. It has been added temporarily to ts_scheduler instead.